bot.py

```diff
@@ -0,0 +1,77 @@
+import heuristics
+import board_heu
+from game import Game, Player, BOARD_SIZE
+from draughts import Board, Move
+import time
+
+"""
+NEW alpha-beta pruning
+
+TO DO:
+- Clean up code
+- For now, everything minimax-related is played on the Board (not Game) 
+  --> change that
+- Problem with moves that are evaluated the same: the model will have a tendency to play a little too much on the left (or right) 
+  --> implement some randomness
+- Minimax takes too long to compute 
+  --> alpha-beta pruning
+- Update alpha-beta pruning in all the files
+
+Minimax handles jump chains as one action.
+"""
+
+def minimax(board: Board, depth: int, alpha: int, beta: int, max_player: bool):
+    """
+    Minimax algorithm
+    Returns the score, the move and the resulting board of the best move to play according to heuristics
+    """
+    if depth == 0 or board.is_over():
+        # return board_heu.piece_count(board), board.move_stack[-1].steps_move, board
+        return board_heu.piece_count(board), board.move_stack[-1]
+    if max_player:
+        max_eval = float('-inf')
+        best_board = None
+        for child in get_children(board):
+            evaluation = minimax(child, depth-1, alpha, beta, False)[0]
```

Comment:
A problem is that we cannot assume that the it is always the other player's turn after playing a move, if we want to be able to handle jump moves correctly! When this is rewritten to use Game objects instead of Board, we could use `game.get_current_player() == Player.WHITE` to find out whether we are maximizing or minimizing